editor/node_3d_editor_gizmos.cpp

```diff
@@ -2979,11 +3418,11 @@ void DecalGizmoPlugin::set_handle(EditorNode3DGizmo *p_gizmo, int p_idx, Camera3
 	Vector3 sg[2] = { gi.xform(ray_from), gi.xform(ray_from + ray_dir * 16384) };
 
 	Vector3 axis;
-	axis[p_idx] = 1.0;
+	axis[p_id] = 1.0;
```

Comment:
Is it an improvement to rename this from idx to id? It seems like this number is the index of the edited axis. Also, if this value is always to going to be an axis index, it should be of the enum type `Vector3::Axis`.

Reply:
I changed all the `p_idx` arguments to `p_id` because now the gizmo system will allow you to define a custom ID for each handle. In this gizmo it's just using the index, but I renamed all the occurrences for consistency.